src/Codeception/Module/Db.php

```diff
@@ -532,18 +532,24 @@ private function readSql($databaseKey = null, $databaseConfig = null): void
      * @return bool|null|string|string[]
      * @throws ModuleConfigException
      */
-    private function readSqlFile(string $filePath): ?string
+    private function readSqlFile(string $configPath): ?string
     {
-        if (!file_exists(Configuration::projectDir() . $filePath)) {
+        if(file_exists($configPath)) {
+           $dumpFile = $configPath;
+        }elseif (file_exists(Configuration::projectDir().$configPath)){
+            $dumpFile = Configuration::projectDir().$configPath;
+        }else {
```

Comment:
Formatting. Please put space after `if` and before `elseif` and `else` keywords.

Please put spaces on both sides of concatenation operator.